README.md

```diff
@@ -302,13 +304,19 @@ After the timeout, the speech-to-text service will automatically stop listening.
 
 #### audio.browserIsSupported(): boolean
 
-Returns true if the browser supports the `browser` speech recognition service.
+Returns `true` if the browser supports the `browser` speech recognition service.
 
 ### Audio Outputs Service
 
-#### outputServicePlay(audio: ArrayBuffer, options: AudioOutputsServicePlayOptions): Promise<void>
+#### audio.initialise()
 
-This plays the generated speech in the message event. Typically, you would want to use this in combination with a `message` conversation handler. You may also wish to pause the microphone while this happens.
+Initialises the audio for characters and media. This method _must_ be called from a user interaction event, such as a click or a keypress. This is due to a security restriction in some browsers.
```

Comment:
I think you also need to mention it _must_ be called before attempting to play any audio or character speech